.github/CONTRIBUTING.md

```diff
@@ -32,14 +32,14 @@ We use GitHub issues to track issues. Report a something by [opening a new issue
 - What actually happens
 - Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
 
-A thorough bug reports is your best change things can be fixed quickly. 
+A thorough bug report is your best change things can be fixed quickly. 
```

Comment:
I still see one lil typo here: change -> chance. After that I'm good for a merge!

Reply:
Sorry I didn't make that clearer in my comment before 🤖